README.md

```diff
@@ -16,7 +16,15 @@
 
 ## TLDR / Quickstart
 
-**Want to install these prompts as slash commands?** Use the [slash-command-manager](https://github.com/liatrio-labs/slash-command-manager) utility to install them in all of your local AI tools:
+### What is this?
+
+This repository provides **structured prompts** (Markdown files) that guide AI assistants through a complete software development workflow: from initial feature idea → detailed specification → task breakdown → implementation → validation. Think of them as reusable playbooks that keep AI agents focused and consistent.
```

Comment:
I would like the "feature idea" to be external in a feature/story etc. and this to be the next level down. Maybe that's in the details....